models/prostate_mri_anatomy/large-files.yaml

```diff
@@ -0,0 +1,3 @@
+large_files:
+  - path: "models/anatomy.pt"
```

Comment:
Please use `model.pt` as the name, we may consume this model in other applications (MONAI label, MONAI deploy, etc.), better to have a common file name.

Thanks.